sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -334,4 +337,41 @@ export class ApplicationControllerService {
 
     return supportingUserDetails;
   }
+
+  /**
+   * Process application in calculation details.
+   * @param studentId student id.
+   * @param programYearId program year id.
+   * @param currentAssessmentId current assessment id.
+   * @return processed object.
+   */
+  async processApplicationInCalculationDetails(
+    studentId: number,
+    programYearId: number,
+    currentAssessmentId: number,
+  ): Promise<ApplicationInCalculationDetails> {
+    const assessmentInCalculationStep =
+      await this.studentAssessmentService.getAssessmentInCalculationStep(
+        studentId,
+        programYearId,
+      );
```

Comment:
This logic is not relevant to the ticket. We do not need to check if the assessment is in calculation step. 

The direction from the technical ACs is to use `studentAssessmentService.getOutstandingAssessmentsForStudentInSequence` to check if there is another outstanding assessment before the assessment corresponding to tracker.